query/src/servers/http/clickhouse_handler.rs

```diff
@@ -85,13 +86,15 @@ async fn execute(
         };
     let mut data_stream = ctx.try_create_abortable(data_stream)?;
     let format_setting = ctx.get_format_settings()?;
-    let mut fmt = OutputFormatType::Tsv;
+    let mut fmt = OutputFormatType::Tsv(HeaderConfig::new(false, false));
     if let Some(format) = format {
         fmt = OutputFormatType::from_str(format.as_str())?;
     }
 
     let mut output_format = fmt.create_format(plan.schema());
+    let header = output_format.serialize_header(&format_setting);
```

Comment:
There are other places we need to call serialize_header.

Eg: download.rs, stage_table.rs

Reply:
get the name from clickhouse
also csv used to call it header

any suggestion?

Reply:
'm mistaken, clickhouse use word `priefix` here,  maybe  it  is more general

Reply:
renamed to prefix